charter.html

```diff
@@ -704,16 +690,34 @@ <h2 id=about>About this Charter</h2>
 
 <script>
 document.addEventListener("DOMContentLoaded", function() {
-  document.querySelectorAll("[id]").forEach(function(heading) {
-    let permalink = document.createElement("a");
-    permalink.href = "#" + heading.getAttribute("id");
-    permalink.className = "permalink";
-    permalink.appendChild(document.createTextNode("#"));
-    if (heading.localName == "tr") {
-      heading = heading.children[0];
+  let sectionCounters = [NaN, 0, 0, 0, 0, 0, 0];
+  document.querySelectorAll("[id]").forEach(function(elementWithID) {
+    const permalink = document.createElement("a");
+    permalink.href = "#" + elementWithID.getAttribute("id");
```

Comment:
I realize that this has been sitting here awhile, but maybe this could be made absolute by starting with `document.querySelector("#about+ul li a").href` (or a simpler query if you tag that element more directly, as I think we should).  Then the permalink is much closer to perma.